src/Sulu/Bundle/WebsiteBundle/Command/DumpSitemapCommand.php

```diff
@@ -93,7 +94,13 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $output->writeln('Start dumping "sitemap.xml" files:');
 
         $webspaceManager = $this->getContainer()->get('sulu_core.webspace.webspace_manager');
+
+        /** @var PortalInformation $portalInformation */
```

Comment:
From what I remember we always  put these comments in the foreach block, don't we?